### PR TITLE
Implemented notifications in frontend

### LINKED ISSUE #294: Generator's validation should be presented to the user
Currently nothing is displayed if an error occurs.

### PR DESCRIPTION
**Pull request by Trinovantes**
Closes #294

### CODE REVIEW

**Comment by Trinovantes** [comment]
4 colours available!
screenshot from 2016-01-29 01 42 00

Slightly tweaked procedure list to match the notifications better
screenshot from 2016-01-29 01 42 49

File: src-backbone/app/js/views/builder/builderNavbarView.js [hunk 1]
```diff
@@ -28,13 +33,12 @@ module.exports = Marionette.ItemView.extend({
         let self = this;
         this.model.destroy({
             success: function() {
-                console.info('Deleted procedure', self.model.get('id'));
+                console.info('Deleted Procedure', self.model.get('id'));
+                Helpers.navigateToDefaultLoggedIn();
```
Comment (Ommy) [inline src-backbone/app/js/views/builder/builderNavbarView.js]: Are we doing notifications for only important things, not every action (i.e., this delete procedure action)

**Comment by Ommy** [comment]
LGTM after comments

File: src-backbone/app/js/views/builder/builderNavbarView.js (hunk 1 above)
Comment (Trinovantes) [inline src-backbone/app/js/views/builder/builderNavbarView.js]: The `console` methods will all be stripped in production. The notifications only show up when there's an error

**Comment by connorcimowsky** [comment]
LGTM

**Pull request merged**